src/main/kotlin/app/revanced/patches/youtube/layout/pivotbar/createbutton/patch/CreateButtonRemoverPatch.kt

```diff
@@ -31,7 +32,7 @@ import app.revanced.patches.shared.settings.impl.SwitchPreference
 @Version("0.0.1")
 class CreateButtonRemoverPatch : BytecodePatch(
     listOf(
-        PivotBarFingerprint
+        PivotBarFingerprint, PivotBarNewFingerprint
```

Comment:
While this works, it is not an ideal solution and should only be done if theres no other way. This fingerprint also contains a fairly large fingerprint, which is why it would be relatively prone to errors. Instead, a fingerprint which works for both, the old and new versions should be created. Since the method which is modified has been proven to change frequently, it is better to resolve to another method, then for example use the `MethodWalker` to get to this method and dynamically find the places which need to be modified by iterating through the instructions or using fingerprints to find the indices of instruction opcode patterns. What I am asking is not too complex, but still might be unclear, which is why if you do not know how to realize this, convert the PR to a draft or close it.